lang/en.ts

```diff
@@ -3453,4 +3425,60 @@ export const lib = {
   oauth: {
     missingClientId: 'Error building oauth URL: missing client ID.',
   },
+  migrate: {
+    componentsToBeMigrated: components =>
+      `The following component types will be migrated: ${components}`,
+    componentsThatWillNotBeMigrated: components =>
+      `[NOTE] These component types are not yet supported for migration but will be available later: ${components}`,
+    errors: {
+      project: {
+        invalidConfig:
+          'The project configuration file is invalid. Please check the config file and try again.',
+        doesNotExist: 'Project does not exist, unable to migrate',
+        multipleApps:
+          'Multiple apps found in project, this is not allowed in 2025.2',
+        alreadyExists: projectName =>
+          `A project with name ${projectName} already exists. Please choose another name.`,
+      },
+      unmigratableReasons: {
+        upToDate: 'App is already up to date',
+        isPrivateApp: 'Private apps are not currently migratable',
+        listedInMarketplace: 'Listed apps are not currently migratable',
+        generic: reasonCode => `Unable to migrate app: ${reasonCode}`,
+      },
+      noAppsEligible: (accountId, reasons) =>
+        `No apps in account ${accountId} are currently migratable${reasons.length ? `\n  - ${reasons.join('\n  - ')}` : ''}`,
+
+      invalidAccountTypeTitle: () =>
+        `${chalk.bold('Developer account not targeted')}`,
+      invalidAccountTypeDescription: (useCommand, authCommand) =>
+        `Only public apps created in a developer account can be converted to a project component. Select a connected developer account with ${useCommand} or ${authCommand} and try again.`,
+      appWithAppIdNotFound: appId =>
+        `Could not find an app with the id ${appId} `,
+      noAppsForProject: 'No apps associated with project {{ projectName }}',
```

Comment:
Should this use the new interpolation pattern?

Reply:
Whoops, great catch.  Will update this